src/main/java/io/cdap/e2e/pages/locators/CdfSysAdminLocators.java

```diff
@@ -51,4 +53,47 @@ public class CdfSysAdminLocators {
 
   @FindBy(how = How.XPATH, using = "//*[@data-cy='Clear All']")
   public static WebElement clearAll;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='wizard-result-icon-close-btn']")
+  public static WebElement clickOnCloseNamespaceWindow;
+
+  public static WebElement namespaceAdded(String nameSpace) {
+    String path = "//*[@href=\"/cdap/ns/" + nameSpace + "/details\"]";
+    return SeleniumDriver.getDriver().findElement(By.xpath(path));
+  }
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(), 'Switch to')]")
+  public static WebElement switchToCreatedNamespace;
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(), 'Go to homepage')]")
+  public static WebElement goToHomepage;
+
+  @FindBy(how = How.XPATH, using = "//a[@href='/cdap/httpexecutor']")  //remove later
```

Comment:
remove this comment in final code Or specify a proper comment

Reply:
In Final PR code while merging this is removed.